manifests/service.pp

```diff
@@ -6,13 +6,13 @@
 class ccs_software::service {
   assert_private()
 
-  $::ccs_software::services.each |String $alias, Array[String] $services| {
+  $ccs_software::services.each |String $alias, Array[String] $services| {
     $services.each |$svc| {
       $epp_vars = {
         desc  => "CCS ${svc} service",
-        user  => $::ccs_software::user,
-        group => $::ccs_software::group,
-        cmd   => "${::ccs_software::ccs_path}/${alias}/bin/${svc}",
+        user  => $ccs_software::user,
+        group => $ccs_software::group,
+        cmd   => "${facts['ccs_software::ccs_path']}/${alias}/bin/${svc}",
```

Comment:
Why does this single instance change to a "facts" reference?

Reply:
Eh, this is a bug in the linter auto-fixer. I'll correct it.